### PR TITLE
Update build to run all available AWS and Azure examples 

### PR DESCRIPTION
**Pull request by stack72**
Also Fixes: #235 	

I am going to work through them section by section to get as many of these enabled as possible

### CODE REVIEW

File: azure-ts-api-management/index.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as azure from "@pulumi/azure";
 
+// use first 10 characters of the stackname as prefix for resource names
+const prefix = pulumi.getStack().substring(0, 9);
+
 // Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
+const resourceGroup = new azure.core.ResourceGroup(`${prefix}-rg`, {
```
Comment (justinvp) [inline azure-ts-api-management/index.ts]: Why is the prefix needed?

File: azure-ts-dynamicresource/index.ts [hunk 2]
```diff
@@ -54,7 +55,7 @@ const cdnEndpoint = new azure.cdn.Endpoint("cdnEndpoint", {
      * For example, the URL for this CDN endpoint when it is created 
      * would be `my-cdn-endpoint.azureedge.net`.
      */
-    name: "my-cdn-endpoint",
+    name: `my-cdn-endpoint-${cdnSuffix}`,
```
Comment (justinvp) [inline azure-ts-dynamicresource/index.ts]: Why is the suffix needed?

File: azure-ts-api-management/index.ts [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+    location: azure.Locations.WestUS2,
```
Comment (mikhailshilkov) [inline azure-ts-api-management/index.ts]: Why specify the location? I thought our general line is to put it into the config?

File: aws-ts-twitter-athena/index.ts [hunk 4]
```diff
@@ -9,6 +9,7 @@ const bucket = new aws.s3.Bucket("tweet-bucket", {
             },
         },
     },
+    forceDestroy: true, //we require this in the example as we are not managing the contents of the bucket via Pulumi
```
[the remark this answers is not in the document]
Comment (stack72) [reply aws-ts-twitter-athena/index.ts]: I agree with this and will bring it down and include in second PR

File: azure-ts-api-management/index.ts [hunk 5]
```diff
@@ -1,8 +1,13 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as azure from "@pulumi/azure";
 
+// use first 10 characters of the stackname as prefix for resource names
```
[the remark this answers is not in the document]
Comment (stack72) [reply azure-ts-api-management/index.ts]: This was actually a copy pasta from another comment :D

File: azure-ts-api-management/index.ts (hunk 1 above)
Comment (stack72) [reply azure-ts-api-management/index.ts]: It's not - this was something I will remove on next PR

File: azure-ts-dynamicresource/index.ts (hunk 2 above)
Comment (stack72) [reply azure-ts-dynamicresource/index.ts]: This test is *all* sorts of broken - I am trying to do something to get it working - I will remove this again

File: azure-ts-api-management/index.ts (hunk 3 above)
Comment (stack72) [reply azure-ts-api-management/index.ts]: This will be removed in next PR

**Pull request merged**